README.md

```diff
@@ -1,5 +1,3 @@
-[![Stories in Ready](https://badge.waffle.io/inaka/cowboy-trails.png?label=ready&title=Ready)](https://waffle.io/inaka/cowboy-trails)
```

Comment:
This whole file was heavily edited, and is probably best viewed in [Preview mode](https://github.com/inaka/cowboy-trails/blob/a66154c3491a0bd0b49f53741959b1fd5cefdbce/README.md).